### PR TITLE
Consistency gaurd

### PR DESCRIPTION
**Pull request by triazo**
Also removes the verification rigor flag, which is currently not being used. (it was always set to full verification)

### CODE REVIEW

**Comment by triazo** [comment]
Will admit that I laughed at the spelling error. Also changed some functions to be part of the consensusset.

File: modules/consensus/consensusset.go [hunk 1]
```diff
@@ -15,38 +15,15 @@ import (
...
-	// being loaded from disk.
-	fullVerification verificationRigor = 0
-
-	// partialVerification indicates that transaction verification can be
-	// skipped. Transaction verification is computationally intensive, and
-	// skipping such a step noticably increases speed when loading many blocks
-	// at once. Usually, partialVerification is used when loading blocks from
-	// disk.
-	partialVerification verificationRigor = 1
-)
```
Comment (DavidVorick) [inline modules/consensus/consensusset.go]: hooray

File: modules/consensus/persist.go [hunk 2]
```diff
@@ -84,7 +97,7 @@ func (cs *ConsensusSet) loadDiffs() {
 		// Blocks loaded from disk are trusted, don't bother with verification.
 		lockID := cs.mu.Lock()
 		// This guard is for when the program is stopped. It is temporary.
-		// DEPRICATED
+		// DEPRECATED
 		if !cs.db.open {
 			break
 		}
```
Comment (DavidVorick) [inline modules/consensus/persist.go]: what makes this code temporary?

File: modules/consensus/validtransaction.go [hunk 3]
```diff
@@ -257,7 +255,9 @@ func (cs *ConsensusSet) TryTransactionSet(txns []types.Transaction) (modules.Con
 	diffHolder.Height = cs.height()
 	defer func() {
```
Comment (DavidVorick) [inline modules/consensus/validtransaction.go]: typically you put the defer after the initalize. IE:

```
startGuard()
defer stopGuard()
```

File: modules/consensus/persist.go (hunk 2 above)
Comment (triazo) [inline modules/consensus/persist.go]: When every structure is loaded from disk immediately, there will be a single write lock on consensus when loading and the database can't be closed while a write lock is held. We'll probably need this guard in gateway though.

**Pull request merged**